.github/workflows/check-release.yml

```
@@ -0,0 +1,15 @@
+on: [push, pull_request]
+name: Check & Release
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
```

I just forgot one thing 😅 Can we use the latest version here?
```suggestion
    - uses: actions/checkout@main
```